components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -0,0 +1,229 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCStatefulRippleView.h"
+#import "private/MDCRippleLayer.h"
+
+static const CGFloat kRippleAlpha = (CGFloat)0.16;
+static const CGFloat kRippleSelectedAlpha = (CGFloat)0.08;
+static const CGFloat kRippleDraggedAlpha = (CGFloat)0.08;
+
+static UIColor *RippleSelectedColor(void) {
+  return [UIColor colorWithRed:(CGFloat)0.384 green:0 blue:(CGFloat)0.933 alpha:1];
+}
```

Comment:
I don’t see any usage of this method: can we delete it?

Reply:
It is used in line 59 and used as the default selection color.

Reply:
Ah, thank you!